cypress/e2e/data-export/display-list-of-settings-pages-permissions.cy.js

```diff
@@ -0,0 +1,50 @@
+import permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import devTeams from '../../support/dictionary/devTeams';
+import testTypes from '../../support/dictionary/testTypes';
+import dataExportPermission from '../../support/fragments/data-export/data-export-permission';
+
+let user;
+
+describe('Data-export', () => {
+  before('Create test data', () => {
+    cy.createTempUser([permissions.uiUsersView.gui, permissions.uiUsersPermissions.gui]).then(
+      (userProperties) => {
+        user = userProperties;
+        cy.login(user.username, user.password);
+        cy.visit(TopMenu.dataExportPath);
```

Comment:
is it better to use login method this way? 
`
cy.login(user.username, user.password, {
          path: TopMenu.dataExportPath,
          waiter: TopMenu.dataExportPath,
        });
`

Reply:
Done.